.ci/test_r_package.sh

```diff
@@ -4,7 +4,7 @@
 R_LIB_PATH=~/Rlib
 mkdir -p $R_LIB_PATH
 echo "R_LIBS=$R_LIB_PATH" > ${HOME}/.Renviron
-echo 'options(repos = "https://cran.rstudio.com")' > ${HOME}/.Rprofile
+CRAN_MIRROR="https://cloud.r-project.org/"
```

Comment:
As we don't write the option into `.Rprofile`, can you please move this declaration to line 4 before `R_LIB_PATH=~/Rlib` to make it easier to find.